src/core/api/thread_api.cpp

```diff
@@ -202,6 +202,12 @@ otError otThreadSetNetworkName(otInstance *aInstance, const char *aNetworkName)
 
     VerifyOrExit(AsCoreType(aInstance).Get<Mle::MleRouter>().IsDisabled(), error = kErrorInvalidState);
 
+#if !OPENTHREAD_CONFIG_ALLOW_EMPTY_NETWORK_NAME
+    // Thread interfaces support a zero length name internally for backwards compatibility, but new names
+    // must be at least one valid character long.
+    VerifyOrExit(nullptr != aNetworkName && aNetworkName[0] != '\0', error = kErrorInvalidArgs);
+#endif // OPENTHREAD_CONFIG_ALLOW_EMPTY_NETWORK_NAME
```

Comment:
```suggestion
#endif 
```
We can remove this comment . Per our style guide we only require it if there are more than 20 lines from the related `#if`.